assets/js/project.js

```diff
@@ -1,7 +1,15 @@
-jQuery( function( $ ) {
-	// $( 'p' ).css( 'background-color', 'blue' );
-});
+$(document).ready(function(){
 
-jQuery( document ).ready( function( $ ) {
-	
-});
\ No newline at end of file
+  //initialize foundation.js
```

Comment:
Be careful of your indentation here.  Anything that's not happening within something else shouldn't look like it is.  For example, the `$(document).foundation();` call isn't at the same spot vertically as your slick init.